athlinks-ontology.ttl

```diff
@@ -6,542 +6,680 @@
+
+:SnowEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/SwimEvent
+
+:SwimEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# 
+# 
+# #################################################################
+# #
+# #    Annotations
+# #
+# #################################################################
+# 
+# 
+
+:AdventureEvent rdfs:label "Adventure Event"@en ;
+	rdfs:comment "An adventure athletic event."@en .
+# 
+
+:AquabikeEvent rdfs:comment "An athletic event comprised of swim and bike activities."@en ;
+	rdfs:label "Aquabike Event"@en .
+# 
+
+:AquathlonEvent rdfs:label "Aquathlon Event"@en ;
+	rdfs:comment "An athletic event comprised of a swim and run."@en .
+# 
+
+:CyclingEvent rdfs:label "Cycling Event"@en ;
+	rdfs:comment "A cycling athletic event."@en .
+# 
+
+:DuathlonEvent rdfs:label "Duathlon Event"@en ;
+	rdfs:comment "An athletic event with two activities."@en .
+# 
+
+:MountainBikeEvent rdfs:comment "A mountain biking athletic event."@en ;
+	rdfs:label "Mountain Bike Event"@en .
+# 
+
+:MultisportHalfIronmanEvent rdfs:label "Half Ironman Event"@en ;
+	rdfs:comment "A half ironman athletic event."@en .
+# 
+
+:MultisportIronManEvent rdfs:comment "An ironman athletic event."@en ;
+	rdfs:label "Ironman Event"@en .
+#
+
+:MultisportOlympicEvent rdfs:comment "An olympic triathlon."@en ;
+	rdfs:label "Olympic Triathlon Event"@en .
+# 
+
+:MultisportSprintEvent rdfs:comment "A sprint triathlon."@en ;
+	rdfs:label "Sprint Triathlon Event"@en .
+# 
+
+:ObstacleEvent rdfs:comment "An obstacle athletic event."@en ;
+	rdfs:label "Obstacle Event"@en .
+# 
+
+:OtherEvent rdfs:comment "An athletic event in an otherwise unspecified category."@en ;
+	rdfs:label "Other Event"@en .
+# 
+
+:Run15kToHalfMarathonEvent rdfs:label "Run 15K to Half Marathon Event"@en ;
+	rdfs:comment "A running event with a distance from 15K up to a half marathon."@en .
+# 
+
+:Run5kTo15kEvent rdfs:comment "A running event from 5 to 15 kilometers in distance."@en ;
+	rdfs:label "Run 5K to 15K"@en .
+# 
+
+:RunHalfToFullMarathonEvent rdfs:label "Half to Full Marathon Event"@en ;
+	rdfs:comment "A running event with a distance from a half to a full marathon."@en .
+# 
+
+:RunMarathonEvent rdfs:comment "A marathon."@en ;
+	rdfs:label "Marathon Event"@en .
+# 
+
+:RunUltraEvent rdfs:comment "A running event with an ultramarathon distance."@en ;
+	rdfs:label "Ultramarathon Event"@en .
+# 
+
+:RunUpTo5kEvent rdfs:label "Run up to 5K Event"@en ;
+	rdfs:comment "A running event up to 5K in distance."@en .
+# 
+
+:SnowEvent rdfs:comment "An athletic event in snow."@en ;
+	rdfs:label "Snow Event"@en .
+# 
+
+:SwimEvent rdfs:label "Swim Event"@en ;
+	rdfs:comment "A swimming athletic event."@en .
+
+# 
+# 
+# 
+# #################################################################
+# #
+# #    General axioms
+# #
+# #################################################################
+# 
+# 
+
+_:genid1 a owl:AllDisjointClasses ;
```

Comment:
I think we don't want this in the ontology? Or if we do we should name it something meaningful.

Reply:
Its an autogenerated blank node identifier. Protege doesn't know how to make nice turtle lists.